docs/cli.md

```diff
@@ -45,7 +45,7 @@ If multiple formats are specified with the same output, only the last is used.
 
 Built-in formatters
 * message - prints the full [`cucumber-messages`](https://github.com/cucumber/cucumber/tree/master/cucumber-messages) object in [varint](https://developers.google.com/protocol-buffers/docs/encoding#varints)-delimited protobuf binary form, which can then be piped to other tools.
-* json - prints the feature as JSON.
+* json - prints the feature as JSON. *Note: this formatter is deprecated and will be removed in the next major release; please migrate to the standalone JSON formatter (see https://github.com/cucumber/cucumber/tree/master/json-formatter)*
```

Comment:
Not sure I agree with this message. We want people to convert to switch to `messages` right? I thought that json formatter only offered a stopgap if its not worth the effort for them? 

Also, think we should get the documentation updated over on json formatter to include where to get the binary and have a cucumber-js example (appears only has a cucumber-ruby example which is still using message binary instead of message ndjson)

Reply:
Yeah, you're right. What about a message like:

> *Note: this formatter is deprecated and will be removed in the next major release. Where you need a structured data representation of your test run, it's best to use the `message` formatter. For legacy tools that depend on the deprecated JSON format, a standalone formatter is available (see https://github.com/cucumber/cucumber/tree/master/json-formatter).*

I'll look at updating the json-formatter docs too.

Reply:
Looks good

Reply:
Duly updated